Servers/WindowServer/WindowServer.ipc

```diff
@@ -74,7 +74,7 @@ endpoint WindowServer = 2
     DismissMenu(i32 menu_id) => ()
 
     AsyncSetWallpaper(String path) =|
-    SetResolution(Gfx::Size resolution) => ()
+    SetResolution(Gfx::Size resolution) => (bool success, int width, int height)
```

Comment:
Please use a `Gfx::Size` instead of separate width and height variables in the response.